packages/bpk-component-image/src/BpkImage.js

```diff
@@ -27,10 +28,28 @@ import STYLES from './bpk-image.scss';
 
 const getClassName = cssModules(STYLES);
 
-const Image = props => {
+type BpkImageProps = {
+  altText: string,
+  height: number,
+  src: string,
+  width: number,
+  className: ?string,
+  inView: ?boolean,
```

Comment:
booleans don't need to be optional - just give them a default prop.

Reply:
Fine, but declaring them optional will make the code self-documenting, I'll remove the `?` anyway